### DISCUSSION TITLE
Activities add-ons for open-day with time management and quotas

### DISCUSSION THREAD

**Discussion by CVZ-es** [pretix/pretix#4676] [Q&A]
Hi, 

I'm trying to setup pretix for our upcoming open-day (free event)
In order to manage the flow of visitors at the entrance, and avoid long queues, we ask people to book for a dedicated time (Eg, entrance at 12h00, 12h15, 12h30, ...) to enter the park and pass security check.

Once entered, during the day we also offer free activities, and we'd like to offer a global booking system with add-on (guided tours, film projections, performance, activities for children, ...)
Each activity has its own quota, and activities take place several time during the day.

The problem I'm facing is when I set quota for activities, I have to link them with an entrance time (eg 12h00).
So when I want to specify the quota, it's for a specific time of entrance.
I can create several variants for an activity, but I don't have a global quota for this activity, not depending of the entrance time of my main event.

To explain in details, somebody can enter at 12h00, and book at the same time for a guided tour at 12h30.
And an other person will book to enter at 12h15, and could also book for the same guided tour at 12h30.
So we should have a global quota for the guided tour at 12h30, not linked with entrance time.

I used add-ons and add-on categories, to setup, maybe I'm wrong, but thanks in advance for the help!

Best
Damien (CVZ-es)

**Comment by raphaelm** [comment]
It's not a supported setup, I fear.

**Reply by CVZ-es** [reply]
Hi Raphael

Thanks for your reply
Best
Damien
